sklearn/gaussian_process/gpc.py

```diff
@@ -0,0 +1,707 @@
+        from the space of allowed theta-values. If greater than 1, all bounds
+        must be finite.
+
+    max_iter: int, optional (default: 100)
+        The maximum number of iterations in Newton's method for approximating
+        the posterior during predict. Smaller values will reduce computation
+        time at the cost of worse results.
+
+    warm_start : bool, optional (default: False)
+        If warm-starts are enabled, the solution of the last Newton iteration
+        on the Laplace approximation of the posterior mode is used as
+        initialization for the next call of _posterior_mode(). This can speed
+        up convergence when _posterior_mode is called several times on similar
+        problems as in hyperparameter optimization.
+
+    copy_X_train : bool, optional (default: False)
+        If True, a persistent copy of the training data is stored in the
+        object. Otherwise, just a reference to the training data is stored,
+        which might cause predictions to change if the data is modified
+        externally.
+
+    random_state : integer or numpy.RandomState, optional
+        The generator used to initialize the centers. If an integer is
+        given, it fixes the seed. Defaults to the global numpy random
+        number generator.
+
+    Attributes
+    ----------
+    X_train_ : array-like, shape = (n_samples, n_features)
+        Feature values in training data (also required for prediction)
+
+    y_train_: array-like, shape = (n_samples,)
+        Target values in training data (also required for prediction)
+
+    classes_ : array-like, shape = (n_classes,)
+        Unique class labels.
+
+    kernel_: kernel object
+        The kernel used for prediction. The structure of the kernel is the
+        same as the one passed as parameter but with optimized hyperparameters
+
+    L_: array-like, shape = (n_samples, n_samples)
+        Lower-triangular Cholesky decomposition of the kernel in X_train_
+
+    pi_: array-like, shape = (n_samples,)
+        The probabilities of the positive class for the training points
+        X_train_
+
+    W_sr_: array-like, shape = (n_samples,)
+        Square root of W, the Hessian of log-likelihood of the latent function
+        values for the observed labels. Since W is diagonal, only the diagonal
+        of sqrt(W) is stored.
+    """
+    def __init__(self, kernel=None, jitter=0.0, optimizer="fmin_l_bfgs_b",
+                 n_restarts_optimizer=1, max_iter=100, warm_start=False,
+                 copy_X_train=False, random_state=None):
+        self.kernel = kernel
+        self.jitter = jitter
+        self.optimizer = optimizer
+        self.n_restarts_optimizer = n_restarts_optimizer
+        self.max_iter = max_iter
+        self.warm_start = warm_start
+        self.copy_X_train = copy_X_train
+        self.random_state = random_state
+        self.rng = check_random_state(self.random_state)
```

Comment:
I will fix this. For the records: it is not leading to a test failure because GPC and GPR due not use self.rng during fit() for the default arguments (only when n_restarts_optimizer > 1).